Obsidian/Net/Packets/Play/Client/EntityVelocity.cs

```diff
@@ -1,12 +1,12 @@
-﻿using Obsidian.Serializer.Attributes;
-using Obsidian.Serializer.Enums;
-using Obsidian.API;
+﻿using Obsidian.API;
 using Obsidian.Entities;
+using Obsidian.Serializer.Attributes;
+using Obsidian.Serializer.Enums;
 using System.Threading.Tasks;
 
 namespace Obsidian.Net.Packets.Play.Client
 {
-    public class EntityVelocity :  IPacket
+    public class EntityVelocity : IPacket
```

Comment:
Can you please next time you make a PR _not_ do this stuff? At least do it in a separate PR because this is hell to review :p